app/renderer/js/utils/dnd-util.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+const ConfigUtil = require(__dirname + '/config-util.js');
+
+function toggle() {
+	const dnd = !ConfigUtil.getConfigItem('dnd', false);
+	let settingList;
```

Comment:
We can re-write this as - 
```javascript
const settingList = ['showNotification', 'silent'];
if (process.platform === 'win32') {
    settingList.push('flashTaskbarOnMessage')
}
```
maybe rename `settingList` to `dndSettingList`, cc @cPhost.